src/app/design/downloads/index.html

```diff
@@ -0,0 +1,12 @@
+<stache 
+  layout="container"
+  pageTitle="Downloads"
+  windowTitle="Downloads">
+  
+<h2 class="sky-section-heading">Sketch UI kit</h2>
+<p>A Sketch file of the SKY UX components. Updated monthly, so check back from time to time to ensure you have the latest and greatest.</p>
+<div>
+    <a href="~/assets/design/skyux-kit.zip">Download the Sketch UI kit</a>
```

Comment:
Do you think it would be useful to make the link standout by making it a button and wrapping it in the "sky-btn sky-btn-default" class?